libraries/AP_VisualOdom/AP_VisualOdom.cpp

```diff
@@ -61,6 +61,11 @@ const AP_Param::GroupInfo AP_VisualOdom::var_info[] = {
 AP_VisualOdom::AP_VisualOdom()
 {
     AP_Param::setup_object_defaults(this, var_info);
+    if (_singleton != nullptr) {
+        gcs().send_text(MAV_SEVERITY_CRITICAL, "BUG: multiple VisualOdometrys not allowed");
```

Comment:
We seem to have various messages and methods when the setup fails.

I see the AC_Fence uses a panic in SITL at least.  I haven't seen the word "BUG:" in the other ones.. wondering if we should try to be consistent with the message.

Reply:
I've changed this to look like Fence.

@magicrub is fixing all of this for us :-)